docs/Other SDL Features/Creating an App Service/index.md

```diff
@@ -651,12 +651,12 @@ sdlManager.addOnRPCRequestListener(FunctionID.GET_APP_SERVICE_DATA, new OnRPCReq
         GetAppServiceData getAppServiceData = (GetAppServiceData) request;
 
         // Send a response
-        GetAppServiceDataResponse response = new GetAppServiceDataResponse();
-        response.setSuccess(true);
-        response.setCorrelationID(getAppServiceData.getCorrelationID());
-        response.setResultCode(Result.SUCCESS);
-        response.setInfo("<#Use to provide more information about an error#>");
-        response.setServiceData(<#Your App Service Data#>);
+        GetAppServiceDataResponse response = new GetAppServiceDataResponse()
+            .setSuccess(true)
```

Comment:
This will not compile because `setSuccess()` is a method from a parent class. This applies to other methods and places in this PR